File-Formats/DocIO/Working-with-Find-and-Replace.md

```diff
@@ -884,6 +884,97 @@ document.Close();
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Find-and-Replace/Replace-misspelled-word).
 
+## Find and replace non-breaking spaces with regular spaces
+
+The following code example illustrates how to find and replace non-breaking spaces with regular spaces in a Word document.
+
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Load an existing Word document.
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+    //Replace all occurrences of non-breaking spaces with regular spaces.
+    document.Replace(ControlChar.NonBreakingSpace, ControlChar.Space, false, false);
+    //Save the Word document.
+    document.Save("Sample.docx", FormatType.Docx);
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Load an existing Word document. 
+Using document As WordDocument = New WordDocument("Input.docx", FormatType.Docx)
+    'Replace all occurrences of non-breaking spaces with regular spaces.
+    document.Replace(ControlChar.NonBreakingSpace, ControlChar.Space, false, False)
+    'Save the Word document.
+    document.Save("Sample.docx", FormatType.Docx)
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//Open the file as stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.input.docx"))
+{
+    //Load the file stream into a Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Replace all occurrences of non-breaking spaces with regular spaces.
+        document.Replace(ControlChar.NonBreakingSpace, ControlChar.Space, false, false);
+        //Save a Word document to MemoryStream.
+        MemoryStream stream = new MemoryStream();
+        await document.SaveAsync(stream, FormatType.Docx);
+        //Save the stream as a Word document file in the local machine.
+        Save(stream, "Sample.docx");
+		//Please refer to the link below to save the Word document in the UWP platform.
```

Comment:
@sharonaumao - Please remove the unwanted spacing from here.

Reply:
@Ramaraj-Marimuthu -I have removed it